config/mapconverter/expandmapconverter/expand.go

```diff
@@ -21,10 +21,10 @@ import (
 	"go.opentelemetry.io/collector/config"
 )
 
-// NewExpandConverter returns a service.ConfigMapConverterFunc, that expands all environment variables for a given config.Map.
+// New returns a config.MapConverterFunc, that expands all environment variables for a given config.Map.
 //
 // Notice: This API is experimental.
```

Comment:
Is this notice still valid after this change?

Reply:
Let's keep changes separately. Not sure that only because we moved the location does this change to not be experimental.